calcrule_social_protection/calculation_rule.py

```diff
@@ -42,23 +40,3 @@ def ready(cls):
                 cls.signal_get_linked_class.connect(cls.get_linked_class, dispatch_uid="on_get_linked_class_signal")
                 cls.signal_calculate_event.connect(cls.run_calculation_rules, dispatch_uid="on_calculate_event_signal")
                 cls.signal_convert_from_to.connect(cls.run_convert, dispatch_uid="on_convert_from_to")
-
-    @classmethod
-    def active_for_object(cls, instance, context, type, sub_type):
-        pass
-
-    @classmethod
-    def check_calculation(cls, instance):
-        pass
-
-    @classmethod
-    def calculate(cls, instance, **kwargs):
-        pass
-
-    @classmethod
-    def get_linked_class(cls, sender, class_name, **kwargs):
-        pass
-
-    @classmethod
-    def convert(cls, instance, convert_to, **kwargs):
-        pass
```

Comment:
After BenefitPlanStrategy will be refactored then these methods could be here and call 
`BenefitPlanStrategyStorage(payment_plan).<method>()` witch actual strategy